### PR TITLE
added missing copyright headers; added spec to verify joining when a seed node is offline

### PR DESCRIPTION
**Pull request by Aaronontheweb**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#### 1.0.9 April 26 2016 ####
```
Comment (Aaronontheweb) [inline RELEASE_NOTES.md]: Ignore. Going to replace in a subsequent commit.

File: src/core/Akka.Cluster.Tests.MultiNode/JoinWithOfflineSeedNodeSpec.cs [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    }
+
+    public class JoinWithOfflineSeedNodeSpecNode1 : JoinWithOfflineSeedNodeSpec { }
+    public class JoinWithOfflineSeedNodeSpecNode2 : JoinWithOfflineSeedNodeSpec { }
+
+    /// <summary>
+    /// Tests to ensure that if we have 2 seed nodes defined and 1 is offline during a <see cref="Cluster.JoinSeedNodes"/>
+    /// command, the joining node will still be marked as <see cref="MemberStatus.Up"/> as long as there are no unreachable nodes
+    /// within the cluster.
+    /// </summary>
```
Comment (Aaronontheweb) [inline src/core/Akka.Cluster.Tests.MultiNode/JoinWithOfflineSeedNodeSpec.cs]: Comment explains the purpose of this spec - tests a common scenario where only 1 seed node needs to be Up in order for a cluster to be joined. Other previous seed nodes can be Downed / offline and the join will still go through correctly.

File: src/core/Akka.Cluster/ClusterEvent.cs [hunk 3]
```diff
@@ -202,6 +202,11 @@ public override int GetHashCode()
                     return hash;
                 }
             }
+
+            public override string ToString()
+            {
+                return $"{GetType()}(Member={Member})";
```
Comment (Aaronontheweb) [inline src/core/Akka.Cluster/ClusterEvent.cs]: Added user-friendly stringifications of all `IClusterDomainEvent`s

File: src/core/Akka.Cluster/Gossip.cs [hunk 4]
```diff
@@ -95,15 +94,6 @@ public Gossip(ImmutableSortedSet<Member> members, GossipOverview overview, Vecto
                 return Overview.Reachability.RemoveObservers(downed.Select(m => m.UniqueAddress).ToImmutableHashSet());
             });
 
-            _reachability = new Lazy<Reachability>(() =>
```
Comment (Aaronontheweb) [inline src/core/Akka.Cluster/Gossip.cs]: Removed. Wasn't used.

File: src/SharedAssemblyInfo.cs [hunk 5]
```diff
@@ -1,8 +1,16 @@
-﻿// <auto-generated/>
+﻿//-----------------------------------------------------------------------
+// <copyright file="SharedAssemblyInfo.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
```
Comment (Silv3rcircl3) [inline src/SharedAssemblyInfo.cs]: This looks like one of the old headers, everywhere else we already set 2016 as year and Lightbend as company name

**Comment by Silv3rcircl3** [comment]
URL is still http://www.typesafe.com, will create another PR to fix it.

**Pull request merged**